### PR TITLE
Downgrade filelock version

### PR DESCRIPTION
**Pull request by KumoLiu**
- Downgrade filelock version since 
https://github.com/Project-MONAI/MONAI/pull/7744
https://github.com/microsoft/nni/blob/767ed7f22e1e588ce76cbbecb6c6a4a76a309805/dependencies/required.txt#L4

### CODE REVIEW

**Comment by SachidanandAlle** [comment]
CI job fails for packaging (it runs integration tests) with an error for cv2

**Comment by tangy5** [comment]
Hi @KumoLiu , it looks like the proposal to remove opencv-python-headless is not working. Can we try another package instead, do you have idea on which package can replace `opencv-python-headless`

File: requirements.txt [hunk 1]
```diff
@@ -28,23 +28,24 @@ pydicom==2.4.4
...
-filelock==3.13.4
+filelock==3.11.0
 passlib==1.7.4
 python-jose[cryptography]==3.3.0
 bcrypt==4.1.2
 shapely==2.0.4
 requests==2.31.0
 requests-toolbelt==1.0.0
 urllib3==2.2.1
+opencv
```
Comment (SachidanandAlle) [inline requirements.txt]: opencv and opencv-python-headless both?

File: setup.cfg [hunk 2]
```diff
@@ -54,13 +54,12 @@ install_requires =
     pydicom-seg==0.4.1
     pynetdicom==2.0.2
     pynrrd==1.0.0
-    opencv-python-headless==4.9.0.80
```
Comment (SachidanandAlle) [inline setup.cfg]: removing this will not help open source monailabel users to do pip-install and run monailabel

**Comment by SachidanandAlle** [comment]
lets go one simple PR first to fix `filelock==3.11.0` version.. and opencv issue can be fixed in another PR accordingly

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**